src/toast/ops/elevation_noise.py

```diff
@@ -181,6 +178,16 @@ def _exec(self, data, detectors=None, **kwargs):
                 view_flags.append(flags)
 
             for det in dets:
+                # If both the A and C values are unset, the noise model is not modified.
+                if self.noise_a is not None:
+                    noise_a = self.noise_a
+                    noise_c = self.noise_c
+                elif "A" in focalplane[det]:
+                    noise_a = focalplane[det]["A"]
```

Comment:
I agree with your comment- let's change these column names to something meaningful.

Reply:
Replaced "A" -> "elevation_noise_a" and "C" -> "elevation_noise_c".